src/PKSim.Core/Model/SupportedProtein.cs

```diff
@@ -0,0 +1,17 @@
+﻿using OSPSuite.Core.Domain;
+
+namespace PKSim.Core.Model;
+
+public class SupportedProtein: IWithName
+{
+   public string Name { get; set; }
```

Comment:
For ine molecule, I need to know the name of the parameter and the name of the ontogeny parameter